main/guides/getting-started/README.md

```diff
@@ -1,163 +1,199 @@
+Until `dapp-game-places` is the default, use `--dapp-base` etc.
+to refer to it:
+
+```sh
+yarn create @agoric/dapp \
+  --dapp-base https://github.com/agoric-labs/ \
+  --dapp-template dapp-game-places \
+  demo
 
-```shell
-node --version # LTS version such as 18.16.0
 ```
 
-**Note:** Agoric will support all long-term support (LTS) versions of Node.js. 
+:::
 
-## Install the Yarn Package Manager
+### Project structure
 
-Follow [Yarn Installation](https://classic.yarnpkg.com/en/docs/install)
-instructions. For example:
+The `demo` directory now has:
 
-```shell
-npm install --global yarn
-yarn --version # 1.22.10 or higher
-```
+- **README.md**
+- **package.json** as usual for a JavaScript project, along with **yarn.lock**
+- **contract/** - smart contract source code, tests, etc.
+- **ui/** - web UI
+- **docker-compose.yml** - for running a local Agoric blockchain
 
-## Install Git
+::: tip TODO: prune api, agstate
 
-Follow [Git installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) or use your platform's package manager.
+Disregard these contents, for now:
 
-```shell
-npm install --global git
-git --version # 2.25.0 or higher
+- **\_agstate**
+- **api**
+  :::
+
+      node_modules
+
+## Install dependencies
+
+The UI depends on the React framework, and the contract depends on
+the Agoric framework. The packages in this project also have
+development dependencies for testing, code formatting, and static analysis.
+
+```sh
+yarn install  # may take several minutes
 ```
 
-## Clone the Agoric SDK
+::: tip Troubleshooting `yarn install`
 
-```shell
-git clone --branch community-dev https://github.com/Agoric/agoric-sdk
-cd agoric-sdk
+If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+
+Also, check that you are on a
+[supported platform](#platform-linux-shell-or-equivalent) and
+not native Windows.
+:::
+
+## Start a local Agoric blockchain
+
+To run a local Agoric blockchain with [docker-compose](https://docs.docker.com/compose/):
+
+```sh
+yarn start:docker
 ```
 
-Cloning and installing the Agoric SDK can take a while. Please be patient.
+::: tip Note: large docker image
 
-## Install NPM Dependencies
+The docker image used here is several gigabytes.
+The fist time you pull it may take several minutes.
 
-```shell
-yarn install
+:::
+
+Look at the logs from the blockchain until it is steadily making blocks:
+
+```sh
+yarn docker:logs
 ```
 
-**Note:** If you run into errors during install or build, make sure you have the relevant developer tools installed. For example, on Debian or Ubuntu Linux, you can run `sudo apt get install build-essential` to install these tools.
+## Deploy the Contract and Start the UI
 
-## Build Packages
+Let's start it up:
 
-```shell
-yarn build
+```sh
+yarn start
```

Comment:
do you plan to wrap all the following commands inside `yarn start` somehow?

Reply:
I hope to, yes:

 - https://github.com/agoric-labs/dapp-game-places/issues/5

Reply:
Does this look interesting approach?

https://github.com/agoric-labs/dapp-game-places/compare/tl-auto-approve?expand=1

Reply:
yes, something like that.

_I avoid shell scripts longer than about 10 lines, but I guess it's the right tool for this job._

Reply:
Due to the way starting the UI clears the screen so that you can't see the mnemonic that you need to add to Keplr, I didn't manage to reduce it to one `yarn start` command.

But I got close: `yarn start:contract` and `yarn start:ui` with keplr setup in between.